crates/quinn_runtime_bevy/Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
-name = "quinn_runtime_bevy"
+authors     = ["The Fish Folk & Spicy Lobster Developers"]
 description = "Quinn runtime implementation built on Bevy's IO task pool."
-version = "0.1.0"
-edition = "2021"
-authors = ["The Fish Folk & Spicy Lobster Developers"]
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/fishfolk/bones"
+edition     = "2021"
+license     = "MIT OR Apache-2.0"
+name        = "quinn_runtime_bevy"
+repository  = "https://github.com/fishfolk/bones"
+version     = "0.2.0"
 
 [dependencies]
-quinn = { version = "0.9.1", default-features = false, features = ["native-certs", "tls-rustls"] }
-quinn-udp = { version = "0.3.0",  default-features = false }
-quinn-proto = { version = "0.9",  default-features = false }
-bevy_tasks = "0.9.1"
-async-executor = "1.4.1"
-async-io = "1.9.0"
-pin-project = "1.0.12"
-futures-lite = "1.12.0"
+async-executor = "1.4"
+async-io       = "1.9"
+bevy_tasks     = "0.10"
+futures-lite   = "1"
+pin-project    = "1"
```

Comment:
Is there a reason to make these version specifications less specific?

I'm not necessarily against it, but I was wondering if it was intentional.

As far as I understand, doing `1` is the same as `1.0.0`, and that would assert that we are compatible with `1.0.0` exactly, and everything after `1.0`.

That's probably true, but in order to be sure, we would have to try and build the game with version `1.0.0` exactly and make sure it still works. I've actually only ever tested the game with whatever was the latest-released version of the crate at the time I added it, so I only knew that the more specific specifier was actually tested.